fern/apis/fern-definition/definition/types.yml

```diff
@@ -1,223 +1,231 @@
   validation: validation.yml
   examples: examples.yml
   source: source.yml
 
-types: 
-  TypeDeclarationSchema: 
+types:
+  TypeDeclarationSchema:
     discriminated: false
-    union: 
+    union:
       - string
       - ObjectSchema
       - EnumSchema
       - DiscriminatedUnionSchema
       - UndiscriminatedUnionSchema
       - AliasSchema
-  
-  # InlinedTypeDeclaration: 
+
+  # InlinedTypeDeclaration:
   #   discriminated: false
-  #   union: 
+  #   union:
   #     - InlinedObjectSchema
   #     - InlinedEnumSchema
   #     - InlinedDiscriminatedUnionSchema
   #     - InlinedUndiscriminatedUnionSchema
-  
-  # InlinedObjectSchema: 
+
+  # InlinedObjectSchema:
   #   extends: ObjectSchema
-  #   properties:   
+  #   properties:
   #     name: string
 
-  # InlinedEnumSchema: 
+  # InlinedEnumSchema:
   #   extends: EnumSchema
-  #   properties:   
+  #   properties:
   #     name: string
-  
-  # InlinedDiscriminatedUnionSchema: 
+
+  # InlinedDiscriminatedUnionSchema:
   #   extends: DiscriminatedUnionSchema
-  #   properties:   
-  #     name: string  
+  #   properties:
+  #     name: string
 
-  # InlinedUndiscriminatedUnionSchema: 
+  # InlinedUndiscriminatedUnionSchema:
   #   extends: UndiscriminatedUnionSchema
-  #   properties:   
-  #     name: string  
+  #   properties:
+  #     name: string
 
-  TypeReferenceSchema: 
+  TypeReferenceSchema:
     discriminated: false
-    union: 
+    union:
       - string
       - TypeReferenceDetailedSchema
-  
-  BaseTypeReferenceSchema: 
-    extends: 
+
+  BaseTypeReferenceSchema:
+    extends:
       - commons.WithDocsSchema
       - commons.WithAvailability
-    properties: 
+    properties:
       default: optional<unknown>
       encoding: optional<encoding.EncodingSchema>
       validation: optional<validation.ValidationSchema>
-  
-  TypeReferenceDetailedSchema: 
-    extends: 
+
+  TypeReferenceDetailedSchema:
+    extends:
       - BaseTypeReferenceSchema
       - commons.WithName
       - commons.WithAudiences
-    properties: 
+    properties:
       type: string
 
-  TypeReferenceDeclarationWithNameSchema: 
+  TypeReferenceDeclarationWithNameSchema:
     discriminated: false
-    union: 
+    union:
       - string
       - TypeReferenceDeclarationWithName
-  
+
   TypeReferenceDeclarationWithName:
-    extends: 
+    extends:
       - BaseTypeReferenceSchema
       - commons.WithName
       - commons.WithAudiences
-    properties: 
+    properties:
       type: string
-  
-  BaseTypeDeclarationSchema: 
-    extends: 
+
+  BaseTypeDeclarationSchema:
+    extends:
       - commons.WithDocsSchema
       - commons.WithAvailability
       - commons.WithAudiences
-    properties: 
+    properties:
       examples: optional<list<examples.ExampleTypeSchema>>
       encoding: optional<encoding.EncodingSchema>
       source: optional<source.SourceSchema>
-  
-  ### Aliases #### 
 
-  AliasSchema: 
+  ### Aliases ####
+
+  AliasSchema:
     extends: BaseTypeDeclarationSchema
```

Comment:
Should `WithInline` just be part of `BaseTypeDeclarationSchema`? It looks like we omitted it from `AliasSchema` which is reasonable, but it's theoretically possible that we generate an alias with a bad name too, and we could just as easily create an in-lined alias type.

That way any TypeDeclaration moving forward will _always_ be able to be in-lined. Curious to hear your thoughts though - I realize this case is less important overall.